com.microsoft.mrtk.graphicstools.unity/Samples~/UnityUI/UnityUI.unity

```diff
@@ -123,6 +123,87 @@ NavMeshSettings:
     debug:
       m_Flags: 0
   m_NavMeshData: {fileID: 0}
+--- !u!1 &3640888
```

Comment:
Let's make a custom Magnifier sample scene rather than use the UnityUI sample scene. Please structure it like the other sample scenes. @Species521 might be able to help?